app/mainwindow2.cpp

```diff
@@ -380,6 +381,13 @@ void MainWindow2::updateSaveState()
     setWindowModified( mEditor->currentBackup() != mBackupAtSave );
 }
 
+void MainWindow2::clearRecentFilesList()
+{
+    mRecentFileMenu->clear();
+    InformativeDialog *informDialog = new InformativeDialog("Notice!", "You have sucessfully cleared the list");
+    informDialog->exec();
```

Comment:
I had this initially and while I too think it's more intuitive, I found it annoying that the computer should decide whether or not that button should be clickable. I can change it though if you are against that.